tasks/utils/reputation.js

```diff
@@ -0,0 +1,17 @@
+const { utils } = require("ethers");
+const initialRepBalances = require("../../inputs/initialRepBalances.json");
+
+const { parseEther } = utils;
+
+const getReputationParams = () => {
+  let repHolders = [];
+  let repAmounts = [];
+  Object.entries(initialRepBalances).forEach(([address, amount]) => {
```

Comment:
Does this sort of simulated a mapping? So you push the values up the stack and then return a key:value pair?

Reply:
The contract expects as input parameters two equally sized arrays with contract addresses and balances. The function generates these arrays from a JSON object.

Reply:
Ahh oke. Thanks

Reply:
And returns both arrays